### DISCUSSION TITLE
Roadmap to Java LTS 11, 13, 17

### DISCUSSION THREAD

**Discussion by thomasjtaylor** [ontop/ontop#565] [General]
Ontop 4.2.1 is built with Java 1.8 (`ontop/pom.xml` `maven.compiler.target`)

I've successfully compiled and ran several of the Ontop unit tests with Java 11 and 13 without modification.

Is there a plan to upgrade to one of the newer Java LTS (long-term-support) releases?

**Comment by ghxiao** [comment]
Hi @thomasjtaylor, 

The next version of Ontop 5.0, planned to be released in December, will drop the support of Java 8, and officially requires Java 11 or 17.
